_templates/README.md

```diff
@@ -0,0 +1,6 @@
+# Templates
+
+This folder has templates for some common types of Handbook pages.
+
+- [Software and tools](./TOOL.md): for any software or tool page, such as GMail, Mural,
```

Comment:
Mind using lowercase "tool.md" instead? I feel like caps are only used for meta documents, like README, CONTRIBUTING, etc but I don't think the templates should fall into that category.

Reply:
Hmm yeah I was thinking along the lines of an ISSUE_TEMPLATE, but I'm happy to make it lowercase.